client/my-sites/email/email-providers-comparison/in-depth/data.tsx

```diff
@@ -45,10 +53,24 @@ export const googleWorkspaceFeatures: EmailProviderFeatures = {
 		/>
 	),
 	list: {
+		access: translate( 'Online & offline access' ),
+		collaboration: translate( 'Real-time collaboration' ),
 		importing: translate( 'Easy to import your existing emails and contacts' ),
-		storage: translate( '30GB storage' ),
+		storage: translate( '30GB storage with shared files' ),
```

Comment:
Maybe this should be `including shared files` for clarity? At a glance, "with shared files" feels ambiguous to me, because I am not sure if the shared files are included in the total or are not counted against the 30GB quota.
Or we could call the storage out as `30GB storage for emails, contacts, and calendar` for Professional Email and `30GB storage for all solutions` for Google?
It might be cleanest to just use `30GB` for both options to keep the comparison simple.

Reply:
I'd vote for keeping this one simple. We already have the learn more links with directions to support. It'd be great to clarify it there with features list. Is anyone actively updating those support guides right now @daledupreez ?

Reply:
I still think we should make clear that Google Workspace also offers files storage. I've updated this label to the following, let me know what you think it:
> 30GB for emails and cloud storage

Reply:
> Is anyone actively updating those support guides right now?

There hasn't been any specific or concerted effort on the respective guides, but we can definitely review them and make/contribute changes.

> 30GB for emails and cloud storage

This looks good to me!